charts/tidb-operator/templates/controller-manager-deployment.yaml

```diff
@@ -53,6 +53,11 @@ spec:
           {{- end }}
           - -tidb-discovery-image={{ .Values.operatorImage }}
           - -cluster-scoped={{ .Values.clusterScoped }}
+          {{- if .Values.clusterPermissions }}
+          - -cluster-permission-node={{ .Values.clusterPermissions.nodes | default false }}
+          - -cluster-permission-pv={{ .Values.clusterPermissions.persistentvolumes | default false }}
+          - -cluster-permission-sc={{ .Values.clusterPermissions.storageclasses | default false }}
```

Comment:
We'd better set the default value to true to keep backward compatibility.

Reply:
default set these args to `true` when `clusterScoped: false`?

Reply:
Sorry, I should go through all the codes before submitting this comment, so the logic is:

- For `clusterscoped: true`, no need to set these three fields
- For `clusterscoped: true`, users have to set the fields explicitly, otherwise, take them as false

In this case, the code here is good.

Reply:
We have to cover the case that if users ever created a tidb-operator with `clusterscoped: false` with cluster-admin privilege, we have to make sure that they can upgrade TiDB Operator successfully and it works as expected (the configurations listed here need to be configured explicitly).
We also have to make it clear in the release notes.

Reply:
added some comments and updated the release notes in PR description.

Reply:
After further thought, I think we still have to set the default value (even if not set) to true to keep backward compatibility because, for all the TiDB Operators installed till this PR, the values for the three fields are true.